packages/react/src/components/button.tsx

```diff
@@ -91,247 +75,119 @@ export function UploadButton<
 ) {
   // Cast back to UploadthingComponentProps<TRouter> to get the correct type
   // since the ErrorMessage messes it up otherwise
-  const $props = props as unknown as UploadButtonProps<TRouter, TEndpoint> &
-    UploadThingInternalProps;
```

Comment:
did we lose these internal overrides? they are needed for theming docs

Reply:
It's just passing the internal flags to the primitives. FYI: I've noticed a couple of the internal overrides are not being handled correctly in the primitive component but I'll fix it now.